linux/cuda_info.bsh

```diff
@@ -681,10 +806,9 @@ function suggested_architectures()
     # the first case first.
     if [ "${CUDA_CODES[$x]:0:1}" == "${arch:0:1}" ]; then
       CUDA_SUGGESTED_CODES+=("${CUDA_CODES[$x]}")
-    else
-      # If you get here, that means you have have a newer card than this version
-      # of cuda supports. So the best answer is to PTX compile for it.
-      CUDA_SUGGESTED_PTX+=("${CUDA_CODES[$x]}")
+    # PTX is for ARCH, not CODES
+    # else
```

Comment:
I would move this comment into the else statement, which is allowed to pass with `:`, and then delete the `CUDA_SUGGESTED_PTX+=("${CUDA_CODES[$x]}")` line